### DISCUSSION TITLE
Wallet connect not working in release mode

### DISCUSSION THREAD

**Discussion by bijosh-heni** [WalletConnect/walletconnect-monorepo#1259] [Developer Support]
Hi,

I can connect to metamask wallet in debug mode using wallet connect library. But in release mode, its not. Any idea anyone?

**Comment by Seifollahy** [comment]
https://cloud.walletconnect.com/app/registry/listing?uuid=ec509a41-c01a-43b0-89a9-1321f5dd7131

**Reply by BenRacicot** [reply]
Do projects need to be authorized in this way? What are you saying with that explorer registry page?

**Comment by hackyguru** [comment]
Hi @bijosh-heni 

Can you let us know a bit more i.e, SDK version, environment info and steps to replicate the issue?
Also, did you check your console for errors?
